kratos/utilities/math_utils.h

```diff
@@ -818,25 +818,7 @@ class MathUtils
  
 	return cross;
     }
-    
-    /**
-     * Identical but it assumes that the output vector is given already the correct size
-     * @param a First input vector
-     * @param b Second input vector
-     * @param c The resulting vector
-     */
-    
-    static inline void CrossProduct(
-        array_1d<double, 3>& c, 
-        const array_1d<double, 3>& a, 
-        const array_1d<double, 3>& b
-        )
-    {
-        c[0] = a[1]*b[2] - a[2]*b[1];
-        c[1] = a[2]*b[0] - a[0]*b[2];
-        c[2] = a[0]*b[1] - a[1]*b[0];
-    }
-    
+
```

Comment:
why did u remove it?

Reply:
just to clarify, it was intended to provide an explicit template instantiation of the  general template before, although most probably it misses the "template<>" in front of it to achieve the objective